src/i18n/en/functions.json

```diff
@@ -118,6 +118,8 @@
     "HOST": "Host",
     "HOST_PATH": "Host path",
     "HUGE": "Huge",
+    "HTTP_TRIGGER_MSG": "You currently didn’t add an HTTP trigger. If you deploy the function without specifying one, Nuclio will add it automatically on deploy with a default configuration. If you want to configure the HTTP trigger differently — &nbsp;<a class=\"link\" data-ng-click=\"$ctrl.addDefaultHttpTrigger()\">add it here</a>&nbsp; and edit it.",
```

Comment:
`&nbsp;` renders as a space character, if you add a space character there are two consecutive spaces there.
```suggestion
    "HTTP_TRIGGER_MSG": "You currently didn’t add an HTTP trigger. If you deploy the function without specifying one, Nuclio will add it automatically on deploy with a default configuration. If you want to configure the HTTP trigger differently —&nbsp;<a class=\"link\" data-ng-click=\"$ctrl.addDefaultHttpTrigger()\">add it here</a>&nbsp;and edit it.",
```